chia/cmds/coin_funcs.py

```diff
@@ -88,7 +88,7 @@ def print_coins(
                 break
             coin, conf_height = coins[i + j]
             address = encode_puzzle_hash(coin.puzzle_hash, addr_prefix)
-            amount_str = print_balance(coin.amount, mojo_per_unit, addr_prefix)
+            amount_str = print_balance(coin.amount, mojo_per_unit, addr_prefix, True)
```

Comment:
it's really difficult to know, at the call-site, what this `True` means. I would highly recommend you make the last parameter named.